katello_38.dot

```diff
@@ -0,0 +1,346 @@
+@startuml
+
+title <b>Satellite BIG Picture</b>
+
+[Browser] as browser
+
+package "Content Host" {
+	[Yum] as yum
+	[RHSM / Subscription Manager] as rhsm
+	[Katello Agent / Goferd] as katello_agent
+	[Puppet Agent] as puppet
+	[Foreman SCAP Client] as foreman_scap_client
+	[Insights] as insights_client
+}
+
+frame "Satellite Server" {
+
+'	interface Waldirio {
+'		Listening on hostname:8080
+'	}
+
+	database "Server DB" {
+		[Postgres\n- Listen on localhost:5432] as postgres
+		[MongoDB\n- Listen on localhost:27017] as mongo_server
+	}
```

Comment:
I'd prefer from a visual standpoint if these two were split and not grouped into a single database entity.